addons/wowl/static/src/services/action_manager/action_manager.ts

```diff
@@ -537,8 +562,10 @@ function makeActionManager(env: OdooEnv): ActionManager {
         return _executeActURLAction(action as ActURLAction);
       case "ir.actions.act_window":
         return _executeActWindowAction(action as ActWindowAction, options);
-      case "ir.actions.act_window_close":
-        return env.bus.trigger("ACTION_MANAGER:UPDATE", { type: "CLOSE_DIALOG" });
+      case "ir.actions.act_window_close": {
+        env.bus.trigger("ACTION_MANAGER:UPDATE", { type: "CLOSE_DIALOG" });
+        return dialogCloseProm;
```

Comment:
This is not true all the time. A close action can be triggered without having a dialog in dom.
A re-render still occurs though because of actionContainer